src/PharoThings-Hardware-Core.package/PotBoard.class/instance/pinAtHeader..st

```diff
@@ -0,0 +1,5 @@
+accessing
+pinAtHeader: anInteger
```

Comment:
I think this PR can be closed. It is covered by recent refactoring.
And this particular API changes are covered by #31 using #gpioHeader idea

Reply:
Yes, I agree. The last refactoring cover theses things.